contrib/helm/event-gateway/values.yaml

```diff
@@ -16,5 +16,5 @@ resources:
   requests:
    cpu: 200m
    memory: 256Mi
-command: ["-db-hosts=eg-etcd-cluster-client:2379", "-log-level=debug"]
-etcd_cluster_name: eg-etcd-cluster
\ No newline at end of file
+command: ["--db-hosts=eg-etcd-cluster-client:2379", "--log-level=debug"]
```

Comment:
What's the reason for this change?

Reply:
I prefer the `GNU` CLI flags and changed these accordingly. In golang now there isn't really any difference between the single `-` and the double `--` when processing long flags. IMHO, short flags are single `-` and long flags are `--`.

https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html

If you're not happy with this change I'm happy to revert.

Reply:
Would be great to be consistent in the docs. Here https://github.com/serverless/event-gateway/blob/cd14c4004bb4aa3c3082526235fff0d8a357736d/docs/running-locally.md e.g. we use single `-`. I'm fine with both options.

Reply:
Also a good call here, will update to keep the double `--` for consistency.